openshift/rh-che.app.yaml

```diff
@@ -0,0 +1,321 @@
+kind: Template
+apiVersion: v1
+metadata:
+  labels:
+    app: che
+  name: che
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    labels:
+      app: che
+    name: che
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      app: che
+    name: che-host
+  spec:
+    ports:
+    - name: http
+      port: 8080
+      protocol: TCP
+      targetPort: 8080
+    selector:
+      app: che
+- apiVersion: v1
+  kind: RoleBinding
+  metadata:
+    labels:
+      app: che
+    name: che
+  roleRef:
+    name: admin
+  subjects:
+  - kind: ServiceAccount
+    name: che
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    labels:
+      app: che
+    name: che-data-volume
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      app: che
+    name: che
+  spec:
+    replicas: 1
+    revisionHistoryLimit: 2
+    selector:
+      app: che
+    strategy:
+      recreateParams:
+        timeoutSeconds: 10000
+      type: Recreate
+    template:
+      metadata:
+        labels:
+          app: che
+      spec:
+        containers:
+        - env:
+          - name: CHE_DOCKER_IP_EXTERNAL
```

Comment:
not sure we still need this one

Reply:
Deployment fails without this var

Reply:
strange. Do you have a specific error ?

Reply:
smth. like ` CHE_DOCKER_IP_EXTERNAL` env var must be set when running on openshift

Reply:
I agree with @davidfestal: that is weird